designs/UI/business_creation.html

```diff
@@ -0,0 +1,110 @@
+                <div class="card-header">Business Registration</div>
+                <div class="card-body">
+                    <form class="">
+                    
+                        <div class="form-group">
+                                <div class="text-left" style="">Business Name</div>
+                                <div class="input-group">
+                                    <div class="input-group-prepend">
+                                    </div>
+                                    <input type="text" class="form-control" placeholder="e.g latifajoy" aria-label="samsonanami" aria-describedby="basic-addon1">
+                                    </div>
+                        </div>
+
+                        <div class="form-group">
+                                <div class="text-left" style="">Location</div>
+                                <div class="input-group">
+                                    <div class="input-group-prepend">
+                                    </div>
+                                    <input type="text" class="form-control" placeholder="e.g Samwel Joyk" aria-label="samsonanami" aria-describedby="basic-addon1">
+                                  </div>
+                        </div>
+
+                        <div class="form-group">
+                                <div class="text-left" style="">Email Address</div>
+                                <div class="input-group">
+                                    <div class="input-group-prepend">
+                                    </div>
+                                    <input type="text" class="form-control" placeholder="e.g samwelemail@gmail.com" aria-label="samsonanami" aria-describedby="basic-addon1">
+                                  </div>
+                        </div>
+
+                        <div class="form-group">
+                                <div class="text-left" style="">Phone number</div>
+                                <div class="input-group">
+                                    <div class="input-group-prepend">
+                                    </div>
+                                    <input type="text" class="form-control" placeholder="e.g +25471709999" aria-label="samsonanami" aria-describedby="basic-addon1">
+                                  </div>
+                        </div>
+                        
+                        <div class="text-left" style="">Select Logo</div>
+                        <div class="form-group">
+                            <div class="input-group">
+                                <input type="file" class="form-control" id="lastname" placeholder="password">
+                            </div>
+                        </div>
+
+                        
+                        <div class="form-group">
+                            <div class="col-sm-offset-2 col-sm-10">
+                              
+                            </div>
```

Comment:
Why do we have an empty markup here?

Reply:
Thank you for that. revisited the issue and polished all the similar cases.